### PR TITLE
Build fixes in fedora (clang 16)

### PR DESCRIPTION
**Pull request by pedroerp**
Summary:
A couple of small build fixes to compile Velox in fedora's toolchain
using clang 16.

Reviewed By: xiaoxmeng

Differential Revision: D56372459

### CODE REVIEW

**Review by majetideepak — APPROVED** [review]

File: velox/common/base/SuccinctPrinter.h [hunk 1]
```diff
@@ -15,6 +15,8 @@
  */
 
 #pragma once
+
+#include <cstdint>
```
Comment (assignUser) [inline velox/common/base/SuccinctPrinter.h:19]: Ah great, this is also a fix for gcc 13 iirc #5898

File: velox/common/process/ProcessBase.h [hunk 2]
```diff
@@ -18,51 +18,36 @@
...
-/**
- * Get current working directory.
- */
+/// Get current working directory.
 std::string getCurrentDirectory();
 
-/**
- * Returns elapsed CPU nanoseconds on the calling thread
- */
+/// Returns elapsed CPU nanoseconds on the calling thread
```
Comment (assignUser) [inline velox/common/process/ProcessBase.h:42]: Just for my illumination what's the difference in those comment types? is /// a doc-string type comment?

**Review by assignUser — APPROVED** [review]

File: velox/common/process/ProcessBase.h (hunk 2 above)
Comment (pedroerp) [reply velox/common/process/ProcessBase.h:42]: Back in the day we had a discussion to converge on the type of comments we use, mostly for consistency, but seems like doxygen understands the three slashes as a special comment type as it generates documentation. 

https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md?plain=1#L119-L137

Comment (pedroerp) [reply velox/common/process/ProcessBase.h:42]: Long story short, three slashes sort of represent documentation; two slashes are just regular comments.

Comment (assignUser) [reply velox/common/process/ProcessBase.h:42]: Thanks!

**Pull request closed without merging**